lib/routes/bupt/news.ts

```diff
@@ -0,0 +1,124 @@
+import { Route } from '@/types';
+import { config } from '@/config';
+import ConfigNotFoundError from '@/errors/types/config-not-found';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+import { CookieJar } from 'tough-cookie';
+import { load } from 'cheerio';
+import { Context } from 'hono';
+import cache from '@/utils/cache';
+import got from 'got';
```

Comment:
`got` is only available dev environment and is treated as deprecated by RSSHub.

Reply:
> `got` is only available dev environment and is treated as deprecated by RSSHub.

What should I use? ofetch doesn't seem to be able to get response headers, let alone handle cookies. Can I use [`fetch-cookies`](https://www.npmjs.com/package/fetch-cookie)?

Reply:
Use ofetch in raw mode like #15140.

Reply:
I see `tough-cookie` in the dependencies, but `ofetch` shouldn't be able to use cookieJar directly, right? Do I have to manage cookies manually?

Reply:
Yes.